packages/contracts/test/voting/multisig.ts

```diff
@@ -630,8 +617,7 @@ describe('Multisig', function () {
 
         expect(await multisig.canApprove(1, signers[0].address)).to.be.false;
 
-        await ethers.provider.send('evm_increaseTime', [2000]);
-        await ethers.provider.send('evm_mine');
+        await advanceTime(2000);
```

Comment:
I would define `startDate` and `endDate` once (e.g. at the top of the file) and then `advanceTimeTo(startDate)` do
`advanceTimeTo(endDate)` so that one does not need to think about if advancing by `2000` moves you past the `startDate` or `endDate`

Reply:
doesn't work all the time... I guess hardhat doesn't reset the chain/block timestamps

Reply:
hm, in my tests it worked but nvm

Reply:
I had some issues with dates and times :sweat: